browser/ui/toolbar/brave_app_menu_model.cc

```diff
@@ -9,9 +9,14 @@
 
 #include "base/notreached.h"
 #include "base/strings/utf_string_conversions.h"
+#include "brave/browser/ui/sidebar/sidebar_service_factory.h"
+#include "brave/browser/ui/sidebar/sidebar_utils.h"
 #include "brave/components/ipfs/buildflags/buildflags.h"
+#include "brave/components/l10n/common/localization_util.h"
+#include "brave/components/sidebar/sidebar_service.h"
```

Comment:
app_menu_model and brave_app_menu_model are `!is_android` so it needs a toolkit_views guard here